blog/models.py

```diff
@@ -0,0 +1,26 @@
+"""This module contains models for the Blog application."""
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Model representing a blog post."""
+
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish post by setting publish datetime."""
```

Comment:
Don't add unnecessary implementation details. A docstring's purpose is not for retelling every bit of the implementation. This should describe what it does, not how.

Reply:
Edited.

Reply:
Marking as resolved.